beacon-chain/sync/metrics.go

```diff
@@ -86,9 +86,13 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "block_arrival_latency_milliseconds",
 			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
-			Buckets: []float64{100, 250, 500, 750, 1000, 1500, 2000, 4000, 8000, 12000, 16000, 20000, 24000},
+			Buckets: []float64{100, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2500, 3000, 3500, 4000, 5000, 6000, 7000, 8000, 9000, 10000, 15000, 20000},
```

Comment:
I'm not comfortable with these aditions. Was the problem that we didn't have granularity or the problem was that we didn't have the right buckets. I would feel much better if we use a logarithmic scale here, but with the same number of ticks.